src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,12 +25,14 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_INSURANCE_POLICY + "Policy ID] "
```

Comment:
maybe block letters? follow the format of the other fields

Reply:
Done